src/main/java/com/thevoxelbox/voxelsniper/brush/type/performer/UnderlayBrush.java

```diff
@@ -24,16 +27,17 @@ public final void handleCommand(String[] parameters, Snipe snipe) {
             if (parameter.equalsIgnoreCase("info")) {
                 snipe.createMessageSender()
                         .message(ChatColor.GOLD + "Reverse Overlay brush parameters:")
-                        .message(ChatColor.AQUA + "d[number] (ex: d3) The number of blocks thick to change.")
-                        .message(ChatColor.BLUE + "all (ex: /b reover all) Sets the brush to affect ALL materials")
+                        .message(ChatColor.AQUA + "/b under d[number] (ex: d3) -- The number of blocks thick to change.")
+                        .message(ChatColor.BLUE + "/b under all -- Sets the brush to affect ALL materials")
+                        .message(ChatColor.BLUE + "/b under some -- Sets the brush to affect natura block types")
                         .send();
                 if (this.depth < 1) {
                     this.depth = 1;
                 }
                 return;
             }
             if (parameter.charAt(0) == 'd') {
-                this.depth = Integer.parseInt(parameter.replace("d", ""));
+                this.depth = Integer.parseInt(parameter.substring(1));
                 messenger.sendMessage(ChatColor.AQUA + "Depth set to " + this.depth);
             } else if (parameter.startsWith("all")) {
                 this.allBlocks = true;
```

Comment:
Putting this here for all following args, missing space between message and variable.